src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -773,12 +767,14 @@ public void selectConstSumPointsOptionsForNewQuestion(String pointsOption) {
         selectConstSumPointsOptions(pointsOption, NEW_QUESTION_NUM);
     }
 
-    public String getGiverTypeForQuestion1() {
-        return giverDropdownForQuestion1.getAttribute("value");
+    public String getGiverTypeForQuestion(int qnNumber) {
+        WebElement giverDropdownForQuestion= browser.driver.findElement(By.id("questionedittext-" + qnNumber));
```

Comment:
Wrong id; you forgot to change after copy-pasting. Also, add a space before `=`.